test/TestProjects/FirstTest-TypeSpec/FirstTest-TypeSpec.tsp

```diff
@@ -111,6 +111,24 @@ model Thing {
 
     @doc("description with xml <|endoftext|>")
     requiredBadDescription: string;
+
+    @doc("optional nullable collection")
+    optionalNullableList?: int32[] | null;
+
+    @doc("required nullable collection")
+    requiredNullableList: int32[] | null;
+}
+
+@doc("A model with a few required nullable properties")
```

Comment:
Why these not in cadl-ranch?

Reply:
I do not know, maybe we could add some

Reply:
We could add some new test cases in cadl ranch